src/Raven.Studio/wwwroot/Content/css/pages/database-settings.less

```diff
@@ -0,0 +1,158 @@
+#database-settings-view {
+    
+    @source-item-color: @database-settings-source-item;
+    @source-item-color-basic: @database-settings-source-item-basic;
+    
+    .all-entries {
+        overflow-x: hidden;
+        top: 50px;
+    }
+    
+    .description-text {
+        width: 500px;
+        word-break: break-word;
+    }
+    
+    .entries-grid {
+        margin-top: 0px;
+    }
+
+    .summary-buttons {
+        margin-right: 30px;
+    }
+    
+    .filter-entries {
+        min-width: 230px;
+    }
+    
+    .add-vertical-split {
+        &::before {
+            content: "";
+            background: @gray-dark;
+            position: absolute;
+            bottom: 15%;
+            left: 0;
+            height: 70%;
+            width: 1px;
+        }
+    }
+    
+    .customized-item {
+        &::after {
+            content: '*';
+            color: @database-settings-customized-item;
+            margin-left: 4px;
+            font-size: large;
+        }
+    }
+    
+    .customized-item-with-error {
+        &::after {
+            content: "\e93f";
+            color: @database-settings-customized-item-error;
+            margin-left: 4px;
+            font-size: small;
+            font-family: 'icomoon' !important;
+        }
+    }
+    
+    .categories-list-container {
+        width: 310px;
+        position: relative;
+    }
+    
+    .edit-entries-titles {
+        padding-left: 10px;
+    }
+    
+    .summary-list-container {
+        position: relative;
+
+        .effective-value {
+            color: darken(@color-1-2, 5%);
+        }
+        
+        .source-item {
+            color: @source-item-color-basic;
+            padding-left: 25px;
+            
+            &::before {
+                font-family: @font-family-icon;
+                font-size: small;
+                margin-left: 5px;
+                padding-left: 10px;
+                color: @source-item-color;
+            }
+            
+            &.null-value::before {
+                font-family: @font-family-icon;
+                font-size: small;
+                margin-left: 5px;
+                padding-left: 10px;
+                color: @source-item-color-basic;
+            }
+        }
+        
+        .source-item-default {
+            &::before {
+                //content: "\e9d9"; // Waiting for icon from RavenDB-14970
+            }
+        }
+
+        .source-item-server {
+            &::before {
+                //content:"\e93d";  // Waiting for icon from RavenDB-14970
+            }
+        }
+
+        .source-item-database {
+            &::before {
+                //content: "\e9ce";  // Waiting for icon from RavenDB-14970
+            }
+        }
+    }
+    
+    .entry-basic-template {
+        .panel-body {
+            padding-top: 0px;
```

Comment:
no need to specify px when value is zero.

Reply:
done